FaunaDB.Client/Client/RuntimeEnvironmentHeader.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Reflection;
+
+namespace FaunaDB.Client
+{
+    internal class RuntimeEnvironmentHeader
+    {
+        private string runtime;
+        private string driverVersion;
+        private string operatingSystem;
+        private string environment;
+
+        private static RuntimeEnvironmentHeader instance;
+
+        private RuntimeEnvironmentHeader() {}
+
+        private void GatherEnvironmentInfo(IEnvironmentEditor environmentEditor)
+        {
+            this.environment = GetRuntimeEnvironment(environmentEditor);
+            this.operatingSystem = GetOperatingSystemName();
+            this.runtime = GetCurrentRuntime();
+            this.driverVersion = Assembly.Load(new AssemblyName("FaunaDB.Client")).GetName().Version.ToString();
```

Comment:
can you use `Assembly.GetExecutingAssembly()`?

Reply:
I tried to use this function at first, but it doesn't work if I build the project against `dotnetstandard1.5`:
```
error CS0117: 'Assembly' does not contain a definition for 'GetExecutingAssembly'
```
This is happening because there is a conditional library, which I think overrides the standard `System.Reflections` one:
https://github.com/fauna/faunadb-csharp/blob/master/FaunaDB.Client/FaunaDB.Client.csproj#L32

I even tried to get rid of the dotnetstandard1.5, but it is specifically used here:
https://github.com/fauna/faunadb-csharp/blob/master/FaunaDB.Client/Types/Encoder.cs#L80-L85
and two more places. It looks like some workaround. I can be mistaken, but it looks like we can create another ticket to avoid overriding `System.Reflection` for dotnetstandard1.5.